terms.html

```diff
@@ -11,6 +11,11 @@
   <dd>
 A set of one or more <a>claims</a> made by an <a>issuer</a>.
 The <a>claims</a> in a credential can be about different <a>subjects</a>.
+<p>
+  Our definition of credential differs from, 
+  <a href="https://csrc.nist.gov/glossary/term/credential">NIST's definitions of credential</a>, 
+  in particular our definition does not require any binding to identity.
```

Comment:
```suggestion
  in particular, our definition does not require any binding to identity.
```

Reply:
What does "identity" mean?

Reply:
I agree with Joe -- we should just strike the "in particular" part, IMO.

Reply:
I think it's not our "identity", here; it's NIST's ["identity"](https://csrc.nist.gov/glossary/term/identity) to which NIST is requiring [their credentials](https://csrc.nist.gov/glossary/term/credential) be bound.

Perhaps ["identity"](https://csrc.nist.gov/glossary/term/identity) should be a data cite in our doc? Or just a link out?

On the other hand, I'm not sure that "our definition does not require any binding to identity" is quite accurate. We require that a credential have a subject, whether URI (explicit) or blank node (implicit), which is, in a sense, a binding to that identifier (in our terminology) which it appears NIST might consider an identity, making this not a difference between their credentials and ours....

Reply:
> On the other hand, I'm not sure that "our definition does not require any binding to identity" is quite accurate.

I'm not sure if that's meant as an interpretation of my comment. If so, that's not quite what I meant, but I do agree that quote would not be accurate, if only because we don't have a formal definition. 

*My* definition of identity would lead to the position that VCs *do* bind to identity because identity is how we recognize, remember, and respond to specific people and things. VCs do that with identifiers and proofs and confidenceMethod.  But we don't require that VCs are bound to "physical and behavioral characteristics" of individuals which is the heart of that NIST definition of Identity.

So, I would argue our definition of identity is what is out of alignment. Hence, my question.

That said, I don't expect that the WG would consider my definition as canon. We certainly haven't defined it in spec text.

I like @dlongley's proposal just to remove the clause "in particular our definition does not require any binding to identity."

Reply:
> What does "identity" mean?
> 
> I don't mean to be obtuse here, but I believe we do bind to identity in that we use identifiers as a means to bind a credential so that we can recognize, remember, and respond to the subject of the VC.
> 
> In contrast, I expect what NIST is doing is a technical "binding" that they believe binds the VC to a particular individual. Which is also a form of identity--and not something VCs require. But to say VCs don't bind identity is incorrect without a specific definition of identity.

I agree with Joe, Dave, and Ted that there is a binding of an identifier with VCs. We are not binding to an individual person, but the unique URL, or a DID, with  a proof or other confidence method. 

I would strike the entire sentence, 

> "in particular, our definition does not require any binding to identity."

Reply:
its gone, I'm resolving this thread.